gui/src/components/Image.js

```diff
@@ -15,21 +15,30 @@ const styles = theme => {
   }
 }
 
-const Image = ({src, width, height, spin, alt, classes}) => (
-  <img
-    src={src}
-    width={width}
-    height={height}
-    className={spin ? classes.animate : ''}
-    alt={alt}
-  />
-)
+const Image = ({src, width, height, spin, alt, classes}) => {
+  const size = {}
+  if (width >= 0) {
```

Comment:
Are these conditionals necessary? Can't `width={width}` and `height={height}` be passed instead? I'm not sure why width & height are being compared to 0's here. This is confusing me because this `<Image/>` component is used by `<Logo/>`. Logo in turn is used in `<Header/>` and *Header never passes height to Logo, it only passes width*